src/views/mfa-verify/InlineTOTPForm.js

```diff
@@ -28,6 +28,9 @@ define(['okta', 'views/shared/TextBox'], function (Okta, TextBox) {
       className: 'button inline-totp-verify',
       title: Okta.loc('mfa.challenge.verify', 'login'),
       click: function () {
+        if(!form.isValid()) {
```

Comment:
Do you need this?

Reply:
Yes, this is not the form button so it doesn't trigger this. The property was actually already required, but was not working because of this reason. @mauriciocastillosilva-okta

Reply:
They do directly `form.model.save();`, skipping the form validation.

Reply:
The factor type is 'token:software:totp', as you can see [here](https://github.com/okta/okta-signin-widget/blob/master/src/models/Factor.js#L174) is already set to required. This is why this is working when I add form validation.

Reply:
I think should be a different PR as this fix is specific to the one-time password.